dbus/methods.go

```diff
@@ -199,6 +199,15 @@ func (c *Conn) GetUnitProperty(unit string, propertyName string) (*Property, err
 	return c.getProperty(unit, "org.freedesktop.systemd1.Unit", propertyName)
 }
 
+// GetServiceProperty returns property for given service name and property name
+func (c *Conn) GetServiceProperty(
```

Comment:
Units and service units are different dbus objects
https://www.freedesktop.org/wiki/Software/systemd/dbus/#serviceunitobjects